### PR TITLE
Update tabs.twig

### PR DESCRIPTION
**Pull request by fryiee**
This updates my change to ensure it covers both methods of slug definition (array key and the 'slug' parameter).

### CODE REVIEW

File: resources/views/form/partials/tabs.twig [hunk 1]
```diff
@@ -2,7 +2,7 @@
     {% spaceless %}
         {% for slug, tab in tabs %}
             <li class="nav-item {{ loop.first ? ' active' }}">
-                <a href="#tab-{{ slug }}" data-toggle="tab" class="nav-link">
+                <a href="#tab-{{ tab.slug ? tab.slug : slug }}" data-toggle="tab" class="nav-link">
```
Comment (Piterden) [inline resources/views/form/partials/tabs.twig]: `<a href="#tab-{{ tab.slug|default(slug) }}" data-toggle="tab" class="nav-link">`

File: resources/views/form/partials/tabs.twig [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 <div class="card-block">
     {% for slug, tab in section.tabs %}
-        <div class="tab-pane {{ loop.first ? ' active' }}" id="tab-{{ slug }}">
+        <div class="tab-pane {{ loop.first ? ' active' }}" id="tab-{{ tab.slug ? tab.slug : slug }}">
```
Comment (Piterden) [inline resources/views/form/partials/tabs.twig]: `<div class="tab-pane{{ loop.first ? ' active' }}" id="tab-{{ tab.slug|default(slug) }}">`

**Review by Piterden — CHANGES_REQUESTED** [review]

File: resources/views/form/partials/tabs.twig (hunk 1 above)
Comment (fryiee) [reply resources/views/form/partials/tabs.twig]: Both acceptable, just depends how much Twig functionality we want to use. Not sure what would actually be quicker to render, @RyanThompson might know.

Comment (Piterden) [reply resources/views/form/partials/tabs.twig]: `{{ tab.slug ?: slug }}` should work too...

**Comment by RyanThompson** [comment]
Attll do! Thanks man!

**Pull request merged**